bioconda_utils/utils.py

```diff
@@ -639,3 +639,26 @@ def get_list(key):
 
     default_config.update(config)
     return default_config
+
+
+def modified_recipes(git_range, recipe_folder):
```

Comment:
No, the globbing actually works fine. I tested it extensively. I think the reason is that git expands it internally. Let's not use shell=True if it is not needed. I agree with `--relative`, though.